pom.xml

```diff
@@ -178,7 +178,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
+      <dependency>
```

Comment:
Is this logically part of this change, or is it a separate work item? If the latter, then can we make it a separate PR?